2006-01-14  Guilhem Lavaux  <guilhem at kaffe.org>
* libraries/clib/sound/alsa/check_constants.h: Check
  for the macro definition because using the assert to check
  the value (needed for alsa-lib-1.0.11-rc2).

Index: kaffe/libraries/clib/sound/alsa/constants_check.h
diff -u kaffe/libraries/clib/sound/alsa/constants_check.h:1.1 kaffe/libraries/clib/sound/alsa/constants_check.h:1.2
--- kaffe/libraries/clib/sound/alsa/constants_check.h:1.1	Wed Apr 28 20:42:58 2004
+++ kaffe/libraries/clib/sound/alsa/constants_check.h	Wed Jan 18 07:37:43 2006
@@ -136,8 +136,12 @@
 	assert(SND_SEQ_PRIORITY_HIGH == org_tritonus_lowlevel_alsa_AlsaSeq_SND_SEQ_PRIORITY_HIGH);
 	assert(SND_SEQ_PRIORITY_MASK == org_tritonus_lowlevel_alsa_AlsaSeq_SND_SEQ_PRIORITY_MASK);
 	assert(SND_SEQ_CLIENT_SYSTEM == org_tritonus_lowlevel_alsa_AlsaSeq_SND_SEQ_CLIENT_SYSTEM);
+#ifdef SND_SEQ_CLIENT_DUMMY
	assert(SND_SEQ_CLIENT_DUMMY == org_tritonus_lowlevel_alsa_AlsaSeq_SND_SEQ_CLIENT_DUMMY);
+#endif
+#ifdef SND_SEQ_CLIENT_OSS
 	assert(SND_SEQ_CLIENT_OSS == org_tritonus_lowlevel_alsa_AlsaSeq_SND_SEQ_CLIENT_OSS);
+#endif
 /* 	assert(SND_SEQ_FILTER_BROADCAST == org_tritonus_lowlevel_alsa_AlsaSeq_SND_SEQ_FILTER_BROADCAST); */
 /* 	assert(SND_SEQ_FILTER_MULTICAST == org_tritonus_lowlevel_alsa_AlsaSeq_SND_SEQ_FILTER_MULTICAST); */
 /* 	assert(SND_SEQ_FILTER_BOUNCE == org_tritonus_lowlevel_alsa_AlsaSeq_SND_SEQ_FILTER_BOUNCE); */
